rocAL/rocAL_pybind/amd/rocal/readers.py

```diff
@@ -0,0 +1,140 @@
+import rali_pybind as b
+from amd.rocal.pipeline import Pipeline
+import amd.rocal.types as types
+
+def coco(*inputs,file_root, annotations_file='', bytes_per_sample_hint=0, dump_meta_files=False, dump_meta_files_path='', file_list='', initial_fill=1024,  lazy_init=False, ltrb=False, masks=False, meta_files_path='', num_shards=1, pad_last_batch=False, prefetch_queue_depth=1,
+                 preserve=False, random_shuffle=False, ratio=False, read_ahead=False,
+                 save_img_ids=False, seed=-1, shard_id=0, shuffle_after_epoch=False, size_threshold=0.1,
+                 skip_cached_images=False, skip_empty=False, stick_to_shard=False, tensor_init_bytes=1048576):
+
+    Pipeline._current_pipeline._reader = "COCOReader"
+    #Output
+    labels = []
+    bboxes = []
+    kwargs_pybind = {"source_path": annotations_file, "is_output":True}
+    b.setSeed(seed)
+    meta_data = b.COCOReader(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
```

Comment:
looks like empty labels and boxes are returned from here which can cause problem in running the pipeline

Reply:
Rajy, This was done to Match DALI's API